### PR TITLE
Add features and fix

### PR DESCRIPTION
**Pull request by Rushaway**
Add Features as ban origin of bans / comms
Imrpove UI 
Fix some text errors
Fix dashboard

### CODE REVIEW

**Review by aXenDeveloper — CHANGES_REQUESTED** [review]
Please use prettier plugin for better formatting code.

**Comment by aXenDeveloper** [comment]
Please remove `center` tag form this PR.

**Comment by Rushaway** [comment]
> Please remove all `center` tag form this PR.

Done.

**Comment by Rushaway** [comment]
https://github.com/aXenDeveloper/sourcebans-web-theme-fluent/pull/50/commits/05b9b08b6e445310f086821c83d8435cf1cd38c5 is fixing https://github.com/aXenDeveloper/sourcebans-web-theme-fluent/issues/51

**Comment by crashzk** [comment]
I believe it can be merged now, right @aXenDeveloper.

@Rushaway I just have a question regarding GameTracker support. If the server is not registered in GameTracker, what happens? Doesn't show anything or shows an error?

**Comment by Rushaway** [comment]
@crashzk It will show a banner of GameTracker who said the IP isn't registered in GameTracker.

**Comment by Caaine** [comment]
not good :S

**Comment by crashzk** [comment]
> @crashzk It will show a banner of GameTracker who said the IP isn't registered in GameTracker.

I understand, I believe that the support for GameTracker is good, my opinion is that it should have it, but give it a way for the user to have control over it in the end, he chooses in the sourcebans options whether to activate it or not, as he has the option to show the admin that banned or not to players.

In any case, it is already a good start to have support for it.

**Comment by Rushaway** [comment]
@crashzk Can be done, but need to edit the core file of Sourcebanspp.
It's a theme feature, not a global support from Sourcebanspp. So will not be implanted until sourcebanspp add this feature too as native.
People are free to remove or comments this code part.

**Comment by crashzk** [comment]
> @crashzk Can be done, but need to edit the core file of Sourcebanspp. It's a theme feature, not a global support from Sourcebanspp. So will not be implanted until sourcebanspp add this feature too as native. People are free to remove or comments this code part.

I see, they've actually done a PR on sourcebans-pp too: https://github.com/sbpp/sourcebans-pp/pull/718

In any case for me even without having this feature I would still use it.

File: sourcebans-web-theme-fluent/page_servers.tpl [hunk 1]
```diff
@@ -35,6 +35,7 @@
                             <tr id="server_{$server.sid}:desc" class="table_hide">
                                 <td colspan="8">
                                     <div class="collapse_content" id="sinfo_{$server.sid}">
+                                    <p class="responsive_show:desktop" align="center"><a href="https://www.gametracker.com/server_info/{$server.ip}:{$server.port}/"><img style="width: auto; height: auto;" src='https://cache.gametracker.com/server_info/{$server.ip}:{$server.port}/banner_560x95.png'/></a></p>
```
Comment (aXenDeveloper) [inline sourcebans-web-theme-fluent/page_servers.tpl:38]: Delete this. Not everyone wants this functionality.

Gametracker is very slow to load which may slow down the page loading speed

**Review by aXenDeveloper — CHANGES_REQUESTED** [review]

**Review by aXenDeveloper — APPROVED** [review]

**Pull request merged**